src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -265,12 +265,12 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 </PropertyGroup>
 
 <Choose>
-	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' != '' And '$(Optimize)' != 'True' ">
+	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' != '' And '$(DebugType)' != 'None' And '$(Optimize)' != 'True' ">
 		<PropertyGroup>
 			<AndroidIncludeDebugSymbols>True</AndroidIncludeDebugSymbols>
 		</PropertyGroup>
 	</When>
-	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' == '' ">
+	<When Condition=" '$(DebugSymbols)' != '' And $(DebugSymbols) And '$(DebugType)' == '' And '$(DebugType)' != 'None' ">
```

Comment:
Re-reading it, this condition doesn't makes sense. If `'$(DebugType)' == ''`, then `$(DebugType)` *cannot* be None, meaning `'$(DebugType)' != 'None'` *must **always** be true*. No?

Meaning this change shouldn't be required *at all*. The change is meaningless.